core/common/src/main/java/alluxio/util/CommonUtils.java

```diff
@@ -852,5 +861,18 @@ public static boolean isAddressReachable(String hostname, int port) {
     return result;
   }
 
+  private static int initMajorVersion() {
```

Comment:
Please add a comment on what this will return. I'm assuming `8` for java 1.8, `11` for java 11 ?